sensors/lidar/lidar_test.go

```diff
@@ -0,0 +1,147 @@
+package lidar_test
+
+import (
+	"context"
+	"errors"
+	"image/color"
+	"testing"
+
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/pointcloud"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/testutils/inject"
+	"go.viam.com/test"
+
+	"go.viam.com/slam/sensors/lidar"
+)
+
+const (
+	testLidarName  = "testLidarName"
+	wrongLidarName = "wrongLidarName"
+)
+
+func TestNew(t *testing.T) {
+	t.Run("Empty sensors array failure", func(t *testing.T) {
```

Comment:
Great test coverage! Can we please have a test for a negative sensor index?

Reply:
Done!